text/0000-variadic-generics.md

```diff
@@ -195,6 +195,15 @@ However, this approach introduces a lot of additional complexity. One of the
 complications is that such a trait couldn't be implemented for `()`, so
 there would have to be separate `Cons` and `Split` traits, rather than one
 unified `Tuple`.
+- Allow partial borrows or tuple reference splitting. Previous proposals have
+included a transformation from `&(Head, Tail...)` to `(&Head, &Tail...)`.
+However, this would require that every tuple `(Head, Tail...)` actually
+contain a sub-tuple `tail...`. This would get in the way of desirable
```

Comment:
IOW, `(Head, Tail...)` would need the same repr `(Head, Tail)`. Not only does this prevent reordering *but forces a specific iteration order for all tuple recursion (and doing the opposite becomes non-trivial).